corehq/apps/app_manager/templates/app_manager/partials/add_new_module_modal.html

```diff
@@ -0,0 +1,101 @@
+{% load i18n %}
+
+<div class="modal fade add-new-module-modal" id="add-new-module-modal" tabindex="-1" role="dialog" aria-labelledby="add-new-module">
```

Comment:
please leave a comment highlighting that this class is **hack**, should not be duplicated, and overrides a lot of the default behavior of its child classes. additionally, it should state that this entire modal should be refactored to use built-in bootstrap 5 utilities.